src/managers/KeybindManager.cpp

```diff
@@ -1358,6 +1358,8 @@ void CKeybindManager::workspaceOpt(std::string args) {
                 }
             }
         }
+    } else if (args == "persistent") {
```

Comment:
workspaceopt is horrible, and on track to be deleted. I'd rather this was a workspace rule.

See `CConfigManager::handleWorkspaceRules` for handling

Reply:
That could work only if we are able to add/remove those rules at runtime, not only at config time.
I can try and code a couple of dispatchers, something like "addworkspacerule" and "removeworkspacerule". 
Would that be ok?

Reply:
@hariseldon78 can't you do it at runtime with `hyprctl keyword workspace`?

e.g.
```
hyprctl keyword workspace "1, monitor:eDP-1"
```

Reply:
exactly as above.

Reply:
ah that's perfect then, I'll do it

Reply:
@vaxerski @Diaoul here it is, I've sent this other pr: #3530. I'll close this one then